### ISSUE TITLE
WSL Shortcuts should use %LOCALAPPDATA%

### ISSUE THREAD

**Issue by MicahZoltu** [mintty/wsltty#60]
The shortcuts created for launching Ubuntu in Mintty appear to have a hard-coded path rather than using the appropriate environment variable.  In particular
```
--rootfs="C:\Users\micah\AppData\Local\Packages\CanonicalGroupLimited.UbuntuonWindows_79rhkp1fndgsc\LocalState/rootfs"
```
should be
```
--rootfs="%LOCALAPPDATA%\Packages\CanonicalGroupLimited.UbuntuonWindows_79rhkp1fndgsc\LocalState/rootfs"
```

**Comment by mintty** [comment]
The target application itself and other parameters (configdir, working dir, icon) are installed thus.
I vaguely remember that this didn't work at this point for some reason. To be revisited, patches welcome.

**Comment by mintty** [comment]
The rootfs is taken from the registry (lxss keys) where it is given in the resolved form, thus the format. Of course it could be substituted explicitly.
Anyway, the next release will not set --rootfs explicitly anymore, so the issue is obsolete.

**Comment by mintty** [comment]
Released 1.8.0.
